src/Mapping/DataTarget/Direct.php

```diff
@@ -138,7 +138,7 @@ protected function checkAssignData($newData, $valueContainer, $getter)
         $currentData = $valueContainer->$getter($this->language);
         DataObject::setHideUnpublished($hideUnpublished);
 
-        $fieldName = lcfirst(str_replace('get', '', $getter));
+        $fieldName = $this->fieldName;
```

Comment:
this is different in case of object bricks. please see https://github.com/pimcore/data-importer/pull/377/files#diff-30298b0c96a44b2c48f3c55e8b8f4c27e8cf26f3177400c8c2ad9761b860cd7eR145